.storybook/components/DesignTokens/Tier2/Shadows.js

```diff
@@ -9,10 +9,7 @@ export class Tier2Shadows extends Component {
       <div>
         <Section title="Box Shadow">
           <Grid>
-            {filterTokens(`eds-theme-box-shadow`).map(function (
-              listItem,
-              index,
-            ) {
+            {filterTokens(`eds-box-shadow-md`).map(function (listItem, index) {
```

Comment:
nit:
```suggestion
            {filterTokens('eds-box-shadow-md').map(function (listItem, index) {
```

Reply:
Removing this since it's only using tier 1 now